src/exercises/compile-and-run/README.md

```diff
@@ -1,30 +1,31 @@
-# Compile and run RISC-V and CHERI-RISC-V programs
+# Compile and run baseline and CHERI-enabled programs
 
 This exercise steps you through getting up and running with code compilation
-and execution for RISC-V and CHERI-RISC-V programs.
+and execution for baseline and CHERI-enabled programs.
 
 The first test program is written in conventional C, and can be compiled
-to RISC-V or CHERI-RISC-V targets:
+to RISC-V, ARMv8, CHERI-RISC-V or Morello targets:
```

Comment:
I wonder if we should use AArch64 rather than Armv8 everywhere, given (a) there are newer versions than v8 (b) 32-bit Armv8 is a thing (it even adds things on top of Armv7) (c) aarch64 is in the triple (d) we use aarch64 for file and target names (cheribuild and ccc)